# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/editing/run/caret-navigation-around-line-break.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/editing/run/caret-navigation-around-line-break.html.ini
--- a/testing/web-platform/meta/editing/run/caret-navigation-around-line-break.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/editing/run/caret-navigation-around-line-break.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,10 +1,11 @@
 [caret-navigation-around-line-break.html]
   [two <span> separators - move up]
     expected:
+      if debug and (os == "linux") and webrender and not fission and not swgl: ["PASS", "FAIL"]
       if not debug and (os == "linux") and webrender: FAIL
+      if not debug and (os == "win"): FAIL
       if not debug and (os == "android"): FAIL
       if not debug and (os == "mac"): FAIL
-      if not debug and (os == "win"): FAIL
       [FAIL, PASS]
 
   [no separator - move up]
@@ -15,17 +16,17 @@
   [<span> separator - move down]
     expected:
       if not debug and (os == "linux") and webrender: FAIL
+      if not debug and (os == "win"): FAIL
       if not debug and (os == "android"): FAIL
       if not debug and (os == "mac"): FAIL
-      if not debug and (os == "win"): FAIL
       [FAIL, PASS]
 
   [two <span> separators - move down]
     expected:
-      if (os == "linux") and webrender and debug and not fission and swgl: ["FAIL", "PASS"]
-      if (os == "linux") and webrender and debug and not fission and not swgl: ["FAIL", "PASS"]
-      if (os == "linux") and webrender and debug and fission: ["FAIL", "PASS"]
-      if (os == "linux") and not webrender and not debug: ["FAIL", "PASS"]
+      if (os == "linux") and (processor == "x86_64") and not swgl and webrender and debug: ["FAIL", "PASS"]
+      if (os == "linux") and (processor == "x86_64") and not swgl and not webrender and not debug: ["FAIL", "PASS"]
+      if (os == "linux") and (processor == "x86_64") and swgl: ["FAIL", "PASS"]
+      if (os == "linux") and (processor == "x86"): ["FAIL", "PASS"]
       FAIL
 
   [<br> separator - move down]
@@ -47,8 +48,8 @@
       if (os == "linux") and (processor == "x86_64") and not swgl and webrender and debug: ["FAIL", "PASS"]
       if (os == "linux") and (processor == "x86_64") and not swgl and not webrender and not debug: ["FAIL", "PASS"]
       if (os == "linux") and (processor == "x86_64") and swgl: ["FAIL", "PASS"]
+      if (os == "linux") and (processor == "x86"): ["FAIL", "PASS"]
       if (os == "mac") and debug: ["FAIL", "PASS"]
-      if (os == "linux") and (processor == "x86"): ["FAIL", "PASS"]
       FAIL
 
   [<wbr> separator - move up]
@@ -56,8 +57,8 @@
       if (os == "linux") and (processor == "x86_64") and not swgl and not webrender and not debug: ["FAIL", "PASS"]
       if (os == "linux") and (processor == "x86_64") and not swgl and webrender and debug: ["FAIL", "PASS"]
       if (os == "linux") and (processor == "x86_64") and swgl: ["FAIL", "PASS"]
+      if (os == "linux") and (processor == "x86"): ["FAIL", "PASS"]
       if (os == "mac") and debug: ["FAIL", "PASS"]
-      if (os == "linux") and (processor == "x86"): ["FAIL", "PASS"]
       FAIL
 
   [<wbr> separator - move down]